src/core/SkipToContentLink/SkipToContentLink.tsx

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/skip-to-content.css';
+
+export type SkipToContentLinkProps = {
+  /**
+   * The id of the main content that the link directs to. Don't forget the #!
+   */
+  href: string;
+  /**
+   * Localize 'Skip to main content' label.
+   * @default 'Skip to main content'
+   */
+  children?: React.ReactNode;
+} & React.ComponentPropsWithoutRef<'a'>;
```

Comment:
`href` should be omitted from the `a` props now that it's being manually included

Reply:
FIxed